package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "auth-i",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon server.js"
```

Comment:
This can just be `"nodemon"` if you specify `server.js` as your value for `"main"` (line 4)